StreetHawk/Classes/Notification/Publish/SHApp+Notification.m

```diff
@@ -224,73 +213,93 @@ - (void)registerForNotificationAndNotifyServer
         {
             [StreetHawk registerOrUpdateInstallWithHandler:nil];
             SHLog(@"Upload notification disable info `revoked` to server.");
+            [StreetHawk tagDatetime: getCurrentLocalDateTime() forKey:@"sh_push_denied"];
```

Comment:
Months ago, server expects ISO time format. The correct behavior is formatting [NSDate date] into ISO format, which has timezone by itself.